mtools/mlaunch/mlaunch.py

```diff
@@ -1128,7 +1130,13 @@ def _start_on_ports(self, ports, wait=False, overrideAuth=False):
                 command_str = re.sub(r'--keyFile \S+', '', command_str)
 
             try:
-                ret = subprocess.check_output([command_str], stderr=subprocess.STDOUT, shell=True)
+                if os.name == 'nt':
+                    # print "command str ", filter(None, command_str.split(' ')), "."
```

Comment:
Could you remove this commented line please? I believe it's a development debug line, so we don't need it anymore. Thanks.